frontend/src/pages/projects/components/NotebookRestartAlert.tsx

```diff
@@ -0,0 +1,13 @@
+import * as React from 'react';
+import { Alert } from '@patternfly/react-core';
+
+const NotebookRestartAlert: React.FC = () => {
+  return (
+    <Alert variant="info" isInline title="Unsaved work will be lost">
```

Comment:
```suggestion
    <Alert variant="info" isInline title="Restarting associated workbenches">
```
🤔  @kywalker-rh  do we know what is more important? The fact that it is restarting or the fact that you could lose data.

What about those moments were we want to tell them we are restarting it _yet again_ in the middle of it starting? Like they add a Storage, add a Data Connection immediately after -- it's likely still restarting because of the new storage.

Reply:
The unsaved work is the biggest issue here. Restarting might be annoying, but the worst outcome would be restarting and losing your work because it wasn't saved.

Reply:
I would think that, if the workbench is in the middle of restarting, the user won't have made any new changes. So in this case I'd think "Restarting associated workbenches" is more important. @kywalker-rh please correct me if I'm wrong :-)

Also, is "Restarting workbench" more accurate for this alert? Unless multiple workbenches are restarting at the same time, "Restarting associated workbenches" might be confusing.

Reply:
My comment was based on the entire concept of having an alert... if this is specifically talking about alerting users after they have already made changes, and we need to restart again, then yes, I agree with @kaedward, sorry for any confusion